CODEOWNERS

```diff
@@ -0,0 +1,96 @@
+# NOTE: "we" in this file will refer to the Compute Infrastructure team at Yelp
+
+# prevent cheeky modifications :)
+CODEOWNERS @Yelp/paasta
+
+## These impact the build process, so we probably want CI review on 'em
+**/Makefile @Yelp/paasta
+Makefile @Yelp/paasta
+# this one is kinda misnamed since it's really a patch/ directory :p
+contrib/ @Yelp/paasta
+tox.ini @Yelp/paasta
+# NOTE: we should probably CODEOWNERS requirements, but this would slow down ML
+# Compute since they have a lot of logic inside service_configuration_lib and
+# tend to make PaaSTA releases that just bump this library
+# XXX: maybe CODEOWNERS to CI + ML Compute?
+
+## These contain entrypoints for PaaSTA or otherwise control entrypoint builds
+setup.py @Yelp/paasta
+debian/control @Yelp/paasta
+debian/paasta-tools.links @Yelp/paasta
+debian/paasta-tools.substvars @Yelp/paasta
+debian/rules @Yelp/paasta
+# this one is unfortunate since operators own the paasta status code for their
+# workloads, but atm all this is in one large file
+paasta_tools/cli/ @Yelp/paasta
```

Comment:
Should we special case spark_run? security_check (is that still used or does that happen elsewhere now)?

Reply:
whoops, err - yea - i'm not quite sure how to exclude things without having N glob rules

...lemme see if a like that looks something like `paasta_tools/cli/spark_run.py` (i.e., no team) will work

Reply:
looks like GH isn't complaining about the lines i added!